components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java

```diff
@@ -138,19 +139,48 @@ public void updateBusinessOwner(BusinessOwner businessOwner) throws AppManagemen
      * @throws AppManagementException
      */
     @Override
-
     public List<BusinessOwner> getBusinessOwners() throws AppManagementException {
         return appMDAO.getBusinessOwners();
     }
 
     /**
+     * Get business owners.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files